### DISCUSSION TITLE
serverpod_auth_server uses a package of an unverified uploader

### DISCUSSION THREAD

**Discussion by andriyndev** [serverpod/serverpod#1041] [Ideas]
"serverpod_auth_server" uses the "email_validator" package which, according to https://pub.dev/packages/email_validator, was uploaded by an unverified uploader. Since serverpod_auth_server is used on the server side, using a package created by an unverified uploader poses a serious security threat for all the Serverpod servers which, as the result, will be at the mercy of the unverified uploader.
Instead, I propose to use a regular expression directly without any additional package or use another package created by a verified uploader such as https://pub.dev/packages/form_validation, https://pub.dev/packages/validation_pro and so on.

**Comment by vlidholt** [comment]
Thanks for letting us know, we'll look into this!
